### PR TITLE
fix Layout login

### PR DESCRIPTION
**Pull request by thiennguyen96841996**

### CODE REVIEW

**Comment by thiennguyen96841996** [comment]
ready

**Comment by thiennguyen96841996** [comment]
ready

**Comment by thiennguyen96841996** [comment]
ready

File: resources/views/auth/login.blade.php [hunk 1]
```diff
@@ -1,50 +1,59 @@
...
+                                    <div class="checkbox font-size-13 d-inline-block p-v-0 m-v-0">
+                                        {{ Form::checkbox('remember', null, false, ['id' => 'remember']) }}
+                                        {{ Form::label('remember', __('Remember me')) }}
                                     </div>
-                                </div> 
+                                    <div class="pull-right">
+                                        <a href="{{ route('password.request') }}">{{ __('Forgot password') }}</a>
+                                    </div>
+                                    <div class="m-t-20 text-right">
+                                        {{ Form::submit(  __('Login') , ['class' => 'btn btn-gradient-success']) }}
```
Comment (BunnyPi04) [inline resources/views/auth/login.blade.php]: convention

File: resources/views/backend/layouts/navbar.blade.php [hunk 2]
```diff
@@ -1,45 +1,45 @@
...
+                    </li>
+                </ul>
+                <ul class="dropdown-menu">
+                    <li>
+                        <a href="#">{{ __('attend') }}</a>
+                    </li>
+                </ul>
+                <ul class="dropdown-menu">
+                    <li>
+                        <a href="#">{{ __('work_month')}}</a>
```
Comment (BunnyPi04) [inline resources/views/backend/layouts/navbar.blade.php]: convention

**Comment by thiennguyen96841996** [comment]
ready

**Comment by BunnyPi04** [comment]
commented

**Pull request merged**